python/triton/runtime/autotuner.py

```diff
@@ -40,7 +41,8 @@ def __init__(
         else:
             self.configs = configs
         self.keys = key
-        self.cache = {}
+        # cache is a mapping of Tuple keys to numbers
+        self.cache: Dict[Tuple, Number] = {}
```

Comment:
1. This is wrong, the cache maps to the selected config
2. The comment restating the type is not helpful.

Reply:
sorry mb, this was an accidental commit